src/Plugins/Parallel.php

```diff
@@ -127,7 +128,10 @@ private function runTestSuiteInParallel(array $arguments): int
             $arguments
         );
 
-        $exitCode = $this->paratestCommand()->run(new ArgvInput($filteredArguments), new CleanConsoleOutput);
+        $input = new ArgvInput($filteredArguments);
+        $isDecorated = $input->getParameterOption('--colors', 'always') !== 'never';
+        $output = new CleanConsoleOutput(ConsoleOutput::VERBOSITY_NORMAL, $isDecorated);
+        $exitCode = $this->paratestCommand()->run($input, $output);
```

Comment:
Main parallel process wasn't passing decoration setting to CleanConsoleOutput. Now checks `--colors` option and passes the decoration setting.